doc/conf.py

```diff
@@ -30,13 +30,11 @@
 
 import sys
 import os
-
 import re
-
 import pip
-
 import subprocess
 
+from shutil import copytree, ignore_patterns, rmtree
```

Comment:
rmtree seems not to be used
```suggestion
from shutil import copytree, ignore_patterns
```